res/values-tr/strings.xml

```diff
@@ -0,0 +1,147 @@
+    <string name="battery_style_title">Pil Stili</string>
+    <string name="battery_style_summary">Pil Simgesi Özelleştirmesi</string>
+    <string name="traffic_options_title">Ağ trafiği göstergeleri</string>
+    <string name="traffic_options_summary">Ağ trafiği bilgilerini görüntüle</string>
+    <string name="misc_extensions_title">Çeşitli ayarlar</string>
+    <string name="misc_extensions_summary"></string>
+
+    <string name="general_tweaks_title">Genel Ayarlar</string>
+    <string name="animations_title">Animasyonlar</string>
+    <string name="buttons_title">Tuşlar</string>
+    <string name="recents_ui_title">Son Kullanılanlar Arayüzü</string>
+    <string name="app_ops_category">App Ops</string>
+    <string name="system_app_remover">Sistem uygulaması kaldırma</string>
+    <string name="dialer_title">Çevirici</string>
+
+    <!-- Color Picker -->
+    <string name="dialog_color_picker">Renk Seçici</string>
+    <string name="press_color_to_apply">Uygulamak için aşağıdaki renge dokunun</string>
+    <string name="arrow_right">→</string>
+    <string name="arrow_down">↓</string>
+    <string name="hex">Hex:</string>
+    <string name="hex_hint">#ff000000</string>
+    <string name="set">Set</string>
+    <string name="color_default">Varsayılan</string>
+
+    <!-- Hardware keys -->
+    <string name="hardware_keys_category">Donanımsal tuşlar</string>
+     <!-- Buttons backlight timeout -->
+    <string name="backlight_timeout_title">Tuş aydınlatması zaman aşımı</string>
+    <string name="backlight_timeout_on">Her zaman etkin</string>
+    <string name="backlight_timeout_3s">3 sn</string>
+    <string name="backlight_timeout_default">5 sn (varsayılan)</string>
+    <string name="backlight_timeout_10s">10 sn</string>
+    <string name="backlight_timeout_15s">15 sn</string>
+    <string name="backlight_timeout_20s">20 sn</string>
+    <string name="backlight_timeout_30s">30 sn</string>
+     <!-- Enable keys backlight-->
+    <string name="button_brightness_title">Tuş aydınlatması</string>
+
+    <!-- Power Menu -->
+    <string name="power_menu_title">Güç menüsü</string>
+    <string name="power_menu_title_summary">Güç menüsü seçeneklerini görüntüleme veya değiştirme</string>
+    <string name="powermenu_screenshot">Ekran görüntüsü</string>
+    <string name="powermenu_power">Güç</string>
+    <string name="powermenu_restart">Yeniden başlat</string>
+    <string name="powermenu_restart_recovery">Gelişmiş yeniden başlat</string>
+    <string name="powermenu_airplane">Uçak modu</string>
+    <string name="powermenu_settings">Ayarlar</string>
+    <string name="powermenu_lockdown">Kilitle</string>
+    <string name="powermenu_soundpanel">Ses Paneli</string>
+    <string name="powermenu_users">Kullanıcılar</string>
+    <string name="powermenu_logout">Çıkış yap</string>
+
+    <!-- HW Keys customization -->
+    <string name="action_value_default_action" translatable="false">default</string>
```

Comment:
@razor84 do not need to add to the localization of the line with translatable="false"